sdfObject/sdfobject-genericlevel.sdf.json

```diff
@@ -124,19 +118,22 @@
         },
         "StepResolution": {
           "description": "Step Resolution, time per step",
-          "enum": [
-            "100 Milliseconds", 
-            "1 Second", 
-            "10 Seconds", 
-            "10 Minutes"
-          ]
+          "sdfChoice": {
+            "100 Milliseconds": {}, 
+            "1 Second": {}, 
+            "10 Seconds": {}, 
+            "10 Minutes": {}
+          }
         },
         "DelayData": {
           "description": "delay in increments of 5mS",
           "type": "number",
+          "unit": "s",
           "minimum": 0,
           "maximum": 255,
-          "multipleOf": 1
+          "multipleOf": 1,
+          "scaleMinimum": 0,
+          "scaleMaximum": 1.275
```

Comment:
What is the difference between scaleMax and max? Seems underspecified in the SDF spec

Reply:
removed the scale numbers since they are not in 1.1